packages/client/components/AnalyticsPage.tsx

```diff
@@ -211,24 +211,35 @@ const AnalyticsPage = () => {
         // See: segmentIo.ts:28 for more information on the next line
         {integrations: {'Google Analytics': {clientId: await getAnonymousId()}}}
       )
-      ReactGA.send({hitType: 'pageview', content_group: getContentGroup(pathname)})
-      amplitude.track(
-        'Loaded a Page',
-        {
-          name: pageName,
-          referrer: makeHref(prevPathname),
-          title,
-          path: pathname,
-          url: href,
-          translated,
-          search: location.search
-        },
-        {
-          user_id: atmosphere.viewerId
-        }
-      )
     }, TIME_TO_RENDER_TREE)
-  }, [isSegmentLoaded, pathname, location.search])
+  }, [isSegmentLoaded, pathname])
+
+  useEffect(() => {
+    ReactGA.send({hitType: 'pageview', content_group: getContentGroup(pathname)})
+  }, [pathname])
+
+  useEffect(() => {
+    const title = document.title || ''
```

Comment:
-1 I think we need the same `setTimeout` hack like we have in l194 because of the comment in l103. Feel free to reorder these to make it easier to find for the next one.
```
"Loaded a Page",
    {
      "name": "Meetings",
      "referrer": "",
      "title": "Meetings | Parabol",
      "path": "/me/tasks",
      "url": "http://localhost:3000/me/tasks",
      "translated": false,
      "search": ""
    },
```